workspaces/tests/deploy_project.rs

```diff
@@ -0,0 +1,27 @@
+#![cfg(feature = "unstable")]
+#![recursion_limit = "256"]
+use test_log::test;
+use workspaces::prelude::*;
+
+#[test(tokio::test)]
+async fn test_dev_deploy_project() -> anyhow::Result<()> {
+    let worker = workspaces::sandbox();
+    let wasm = workspaces::compile_project("./tests/test-contracts/status-message").await?;
+    let contract = worker.dev_deploy(&wasm).await?;
+
+    let _res = contract
+        .call(&worker, "set_status")
+        .args_json(("foo",))?
+        .gas(300_000_000_000_000)
```

Comment:
could replace this to `.max_gas()` now that https://github.com/near/workspaces-rs/pull/94 has made it in